build/yamls/antrea-eks.yml

```diff
@@ -375,6 +375,8 @@ data:
 
     # Enable metrics exposure via Prometheus. Initializes Prometheus metrics listener.
     #enablePrometheusMetrics: false
+    # Enable flow exporter that exports IPFIX flow records for Antrea flows from conntrack module.
+    #enableFlowExporter: false
```

Comment:
how confident are we in this configuration parameter? what are we exporting the records to (how to configure the destinations)?.

Reply:
When the support for sending IPFIX records to collector is added, we will have flowCollector config parameter that takes in IP+port. This could be redundant then.

Reply:
Do you think we should extend the comment to specify that this config parameter may go away in the future? @jianjuns